edf/FindForm.cs

```diff
@@ -4,15 +4,24 @@
 using System.IO;
 using System.Reflection;
 using System.Windows.Forms;
+using System.Text.Json;
```

Comment:
Could you try to migrate to the NewtonsoftJSON library, this is the preferred choice in our case.

Reply:
Discussed. Let's leave this library.

Reply:
Native solution.
Also - https://samiprogramming.medium.com/system-text-json-vs-newtonsoft-json-d01935068143